app/lib/features/chat/pages/room_page.dart

```diff
@@ -118,18 +118,19 @@ class _RoomPageState extends State<RoomPage> {
   }
 
   Widget avatarBuilder(String userId) {
+    var avtr = roomController.getUserAvatar(userId);
+
     return Padding(
       padding: const EdgeInsets.only(right: 10),
       child: SizedBox(
         height: 28,
         width: 28,
-        child: CustomAvatar(
-          uniqueKey: userId,
-          avatar: roomController.getUserAvatar(userId),
+        child: ActerAvatar(
+          mode: DisplayMode.User,
+          uniqueId: userId,
+          avatarProviderFuture: avtr != null ? remapForAvatar(avtr) : null,
           displayName: roomController.getUserName(userId),
-          radius: 50,
-          isGroup: false,
-          stringName: simplifyUserId(userId)!,
+          size: 50,
```

Comment:
Is the cache size provided in ```ActerAvatar``` or we are fetching images from rust via thumbnails.